packages/expo/src/Expo.fx.tsx

```diff
@@ -7,20 +7,11 @@ import 'expo-asset';
 import Constants, { ExecutionEnvironment } from 'expo-constants';
 import * as Font from 'expo-font';
 import { NativeModulesProxy, Platform } from 'expo-modules-core';
-import React from 'react';
-import ReactNative, { AppRegistry, ErrorUtils, StyleSheet } from 'react-native';
+import ReactNative, { StyleSheet } from 'react-native';
 
-import DevAppContainer from './environment/DevAppContainer';
 import { createErrorHandler } from './errors/ExpoErrorManager';
 import { createProxyForNativeModules } from './proxies/NativeModules';
 
-// Represents an app running in the store client or an app built with the legacy `expo build` command.
-// `false` when running in bare workflow, custom dev clients, or `eas build`s (managed or bare).
-// This should be used to ensure code that _should_ exist is treated as such.
-const isManagedEnvironment =
-  Constants.executionEnvironment === ExecutionEnvironment.Standalone ||
```

Comment:
Since Classic Builds are no longer supported for SDK 47+, removing `ExecutionEnvironment.Standalone` + updating docs would make for a good follow-up PR.